src/main/java/uk/gov/hmcts/divorce/document/print/LetterPrinter.java

```diff
@@ -117,21 +118,16 @@ private Document documentFromLetter(Letter letter) {
 
     private List<Letter> addAnyAttachmentsToPackForGeneralLetter(final CaseData caseData) {
 
-        ListValue<GeneralLetterDetails> generalLetterDetailsListValue = firstElement(caseData.getGeneralLetters());
-
-        List<Letter> attachmentLetters = new ArrayList<>();
-
-        if (generalLetterDetailsListValue != null) {
-
-            GeneralLetterDetails letterDetails = generalLetterDetailsListValue.getValue();
-
-            List<ListValue<Document>> documents = letterDetails.getGeneralLetterAttachmentLinks();
-
-            if (!CollectionUtils.isEmpty(documents)) {
-                attachmentLetters = mapToLetters(documents, GENERAL_LETTER);
```

Comment:
If we're no longer using the mapToLetters() DocumentUtil method, could it be deleted?

Reply:
Removed unused method and unit test